### ISSUE TITLE
[Feature Idea] Scalebar that adjust on zoom

### ISSUE THREAD

**Issue by lupemba** [MakieOrg/Tyler.jl#88]
It would be nice to have a scalebar that adjust on zoom so you easy can have a length reference in the figure. 
The syntax could be similar to `axislegend` so it can be added with a single line of code and the position and style can be set via arguments.

![image](https://github.com/MakieOrg/Tyler.jl/assets/30597266/65a7f03c-afde-480f-8bbf-db7e134c346a)

**Comment by lazarusA** [comment]
Maybe give a go with a PR :) 
@jkrumbiegel showed a way of doing it here:

 https://discourse.julialang.org/t/scale-division-in-makie/105650/2

**Comment by lupemba** [comment]
> Maybe give a go with a PR :) @jkrumbiegel showed a way of doing it here:
> 
> https://discourse.julialang.org/t/scale-division-in-makie/105650/2

Yes exactly like that.  I will put it on my list and give it a go when I have some time :)

**Comment by asinghvi17** [comment]
I think https://github.com/JuliaAPlavin/MakieExtra.jl has something like this now.  I guess there should also be an option to turn it off...
